kotlin/hexagon-netty-epoll/build.gradle

```diff
@@ -0,0 +1,31 @@
+
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.6.+"
+}
+
+ext {
+    hexagonVersion = "2.0.7"
+    nettyVersion = "4.1.76.Final"
```

Comment:
I'm pinning that version because is the one that the Hexagon adapter uses, and tests in the framework are run against that version. So for me it would be the safer option to ensure everything runs as previously tested.